tests/jsx_quotes/test.js

```diff
@@ -0,0 +1,6 @@
+<SimpleComponent
+  str0="abc"
+  str1='abc'
+  str2="a'b'c"
+  str3='a"b"c'
```

Comment:
```jsx
  str4="a&quot;b&quot;c"
  str5='a&apos;b&apos;c'
```

Reply:
Not sure why Github is still showing this as outstanding but the change has been made.

Reply:
Because the comment was made above the changes.. :)

Reply:
Yeah, I would have just expected it to go away after the changes were made. See: [here](https://github.com/baelec/prettier/blob/428d0e772cb01d344595c92c5768a9843520b640/tests/jsx_quotes/test.js).

Reply:
GitHub only collapses comments when the line the comment is on is changed.

Don’t worry about it too much, it’s not a problem :)